lib/api-base/authorizer.ts

```diff
@@ -33,11 +34,13 @@ import { Secret } from 'aws-cdk-lib/aws-secretsmanager';
  * @property {Layer} authorizerLayer - Lambda layer for authorizer lambda.
  * @property {IRole} role - Execution role for lambdas
  * @property {ISecurityGroup[]} securityGroups - Security groups for Lambdas
+ * @property {Map<number, ISubnet>} importedSubnets for Lambdas
  */
 type AuthorizerProps = {
     role?: IRole;
-    vpc?: IVpc;
+    vpc?: Vpc;
     securityGroups?: ISecurityGroup[];
+    importedSubnets?: Map<number, ISubnet>;
```

Comment:
Is this being used indirectly somewhere? I could have missed it but I don't see it being used in this commit.